contracts/VestingEscrow.vy

```diff
@@ -0,0 +1,348 @@
+# @version 0.3.7
+
+"""
+@title Vesting Escrow
+@author Curve Finance, Yearn Finance, Lido Finance
+@license MIT
+@notice Vests ERC20 tokens for a single address
+@dev Intended to be deployed many times via `VotingEscrowFactory`
+"""
+
+from vyper.interfaces import ERC20
+
+
+event Activated:
+    recipient: indexed(address)
+    amount: uint256
+    owner: address
+    manager: address
+
+
+event Claim:
+    recipient: indexed(address)
+    beneficiary: address
+    claimed: uint256
+
+
+event RevokeUnvested:
+    recipient: indexed(address)
+    revoked: uint256
+
+
+event OwnerChanged:
+    owner: address
```

Comment:
I think it's quite common to include the old value in an event as well.

Reply:
You can always fetch old event